docs/src/concepts/channels/latest.md

```diff
@@ -1,29 +1,23 @@
-# Channels
+# latest
 
-`fuelup` adopts a simplified version of `rustup` [channels](https://rust-lang.github.io/rustup/concepts/channels.html). Currently, only the `latest` channel will be published and serve as a source of distribution of Fuel toolchain binaries.
+### `latest`
```

Comment:
```suggestion
## `latest`
```

My `markdown` linter, complains about header levels not increased one by one Although I am not sure if this is visually acceptable 👀